src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -1,31 +1,34 @@
 /* eslint-disable jsx-a11y/aria-role */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
 import React, { useContext, PropsWithChildren } from 'react';
 import pathOr from 'ramda/src/pathOr';
 import { ServiceContext } from '../../../contexts/ServiceContext';
 import VisuallyHiddenText from '../../../legacy/psammead/psammead-visually-hidden-text/src';
+import BylineCss from './index.styles';
+import { RightChevron } from '../../../components/icons';
 import {
-  Author,
-  JobRole,
-  BylineList,
-  LineBreak,
-  StyledBylineSection,
-} from './index.styles';
+  getBodyCopy,
+  getBrevier,
+} from '../../../legacy/psammead/gel-foundations/src/typography';
+import { getSansBold } from '../../../legacy/psammead/psammead-styles/src/font-styles';
 
 type Props = {
   blocks: any;
 };
 
 const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
-  const { script, service, translations } = useContext(ServiceContext) as {
+  const { service, script, translations } = useContext(ServiceContext) as {
```

Comment:
The `as { }` part can now be removed as `ServiceContext` is fully typed.

Reply:
Done